VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -263,11 +272,27 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     private void updateUIForOrientation(){
-        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
+
+        Fragment fragment = null;
+        int currentItem = pager.getCurrentItem();
+
+        if (currentItem < pagerAdapter.getCount()) {
+            fragment = pagerAdapter.getItem(currentItem);
```

Comment:
This instantiates a new fragment, which we then discard. That seems a pretty roundabout way to see if the selected unit is a video.

These 9 lines could be replaced with 1:
`final boolean isCurrentFragmentVideoUnit  = selectedUnit instanceof VideoBlockModel &&
                ((VideoBlockModel) selectedUnit).getData().encodedVideos.getPreferredVideoInfo() != null`

Granted, we could move that check to a method so that the logic is shared between this and the implementation of `pagerAdapter.getItem()`